### PR TITLE
Gdr 2638

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Related JIRA issue: GDR-2638
  
## Why was it changed?
To properly import public PRISM data
  
# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: R/prism_to_gdrDF.R [hunk 1]
```diff
@@ -102,6 +102,10 @@ convert_LEVEL6_prism_to_gDR_input <- function(prism_data_path,
   cell_lines <- data.table::fread(cell_line_data_path)
   treatment <- data.table::fread(treatment_data_path)
   res <- data.table::fread(prism_data_path)
+
```
Comment (j-smola) [inline R/prism_to_gdrDF.R:105]: Is column "V1" also present In public PRISM data? 👀

File: R/prism_to_gdrDF.R [hunk 2]
```diff
@@ -123,24 +127,29 @@ convert_LEVEL6_prism_to_gDR_input <- function(prism_data_path,
...
   full_data <- merge(full_data,
-                     treatment[, c("column_name", "broad_id", "name", "dose", "moa")],
+                     treatment[, intersect(names(treatment),
+                                           c("column_name", "broad_id", "name", "dose", "moa")),
+                               with = FALSE],
                      all.x = TRUE,
                      by = "column_name")
 
   full_data$value <- pmin(readout_min, 2 ^ full_data$value)
+  untrt_tag <- gDRutils::get_env_identifiers("untreated_tag")[1]
```
Comment (j-smola) [inline R/prism_to_gdrDF.R:137]: 👍 


(`"vehicle"` is characteristic for PRISM?)

**Review by j-smola — APPROVED** [review]
Wouldn't it be a good idea to add some tests?

File: R/prism_to_gdrDF.R (hunk 2 above)
Comment (bczech) [reply R/prism_to_gdrDF.R:137]: what do you mean characteristic? In gDR we use `vehicle` to indicate untreated controls

File: R/prism_to_gdrDF.R (hunk 1 above)
Comment (bczech) [reply R/prism_to_gdrDF.R:105]: Yes, when it's represented by matrix

File: R/prism_to_gdrDF.R (hunk 2 above)
Comment (j-smola) [reply R/prism_to_gdrDF.R:137]:
```
> gDRutils::get_env_identifiers("untreated_tag")
[1] "vehicle"   "untreated"
```

And `"untreated"`?

Comment (bczech) [reply R/prism_to_gdrDF.R:137]: Yes, we use either "vehicle" or "untreated" to indicate untreated conditions

**Pull request merged**